crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -284,11 +284,15 @@ impl StateStore for MemoryStore {
 
     async fn save_changes(&self, changes: &StateChanges) -> Result<()> {
```

Comment:
Instead of the manual `save changes:` prefix in every debug/trace statement, you can add an instrument derive here that will have the same effect:

```suggestion
    #[instrument(skip(self, changes))]
    async fn save_changes(&self, changes: &StateChanges) -> Result<()> {
```

If you remove "self" or "changes" from the `skip` list, they will be logged on every debug/trace statement too.

Reply:
Done, I think.